code/modules/research/design_disks/engineering.dm

```diff
@@ -0,0 +1,110 @@
+/*
+	This file contains disks with engineering designs.
+*/
+
+// ENGINEERING EQUIPMENT / PARTS
+
+/obj/item/disk/design_disk/engineering/ //Engineering parent type
+	name = "design disk - Engineering"
+	desc = "A design disk containing engineering equipment."
+	color = "#eece17"
+	illustration = "hammer"
+	max_blueprints = 1
+
+/obj/item/disk/design_disk/engineering/Initialize() //wip- not sure what else to put here
+	. = ..()
+	blueprints[1] = new /datum/design/inducer
```

Comment:
slightly advanced welder, t-ray, maybe even a hardhat? so tempted to just say remove all the designs and move over to this system but that's out of scope